tools/plist_to_html/plist_to_html/PlistToHtml.py

```diff
@@ -220,6 +223,16 @@ def create_statistics_html(self, output_dir):
         Creates an statistics.html file which contains statistics information
         from the HTML generation process.
         """
+        def severity_order(severity):
+            """
+            This function determines in which order severities should be
+            printed to the output. This function can be given via "key"
+            attribute to sort() function.
+            """
+            severities = ['CRITICAL', 'HIGH', 'MEDIUM', 'LOW', 'STYLE',
+                          'UNSPECIFIED']
+            return severities.index(severity)
+
```

Comment:
I vaguely remember this exact enum existing somewhere in the core CodeChecker library... do we want to duplicate this, creating an additional maintenance point?

Reply:
the plist to html too is designed to be independent and self contained should not depend on any other parts